x-pack/plugins/saved_objects_tagging/public/management/actions/bulk_delete.ts

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { OverlayStart, NotificationsStart } from 'src/core/public';
+import { i18n } from '@kbn/i18n';
+import { ITagInternalClient } from '../../tags';
+import { TagBulkAction } from '../types';
+
+interface GetBulkDeleteActionOptions {
+  overlays: OverlayStart;
+  notifications: NotificationsStart;
+  tagClient: ITagInternalClient;
+  setLoading: (loading: boolean) => void;
+}
+
+export const getBulkDeleteAction = ({
+  overlays,
+  notifications,
+  tagClient,
+  setLoading,
+}: GetBulkDeleteActionOptions): TagBulkAction => {
+  return {
+    id: 'delete',
+    label: i18n.translate('xpack.savedObjectsTagging.management.actions.bulkDelete.label', {
+      defaultMessage: 'Delete',
+    }),
```

Comment:
It seems like adding a `count` here would require rearchitecting a little so I didn't dig too much into it. Is adding a `count` something we could feasibly do? (It's not super critical but would be nice)

```suggestion
    label: i18n.translate('xpack.savedObjectsTagging.management.actions.bulkDelete.label', {
      defaultMessage: 'Delete',
    }),
    'aria-label': i18n.translate('xpack.savedObjectsTagging.management.actions.bulkDelete.label', {
      defaultMessage: 'Delete {count} tags',
      values: { count },
    }),
```

Reply:
We already have the `count` info in the action, so changing the button label would be trivial. Would just like to have confirmation from @MichaelMarcialis as the current wording follows the provided design.

Reply:
Would a aria-label such as `Delete selected tags` be enough though? It would avoid changing the action construction from within the action bar.

Reply:
`Delete selected tags` is better than `Delete` but less good than `Delete {count} tags`